src/main/scala/util/DFT.scala

```diff
@@ -0,0 +1,27 @@
+// See LICENSE for license details.
+package sifive.blocks.util
+
+import chisel3._
+
+import freechips.rocketchip.config.{Parameters}
+import freechips.rocketchip.diplomacy._
+
+trait HasDFT[T <: Data] {
+  
+  def dftNode: BundleBridgeSource[T]
+
+  def makePort(implicit p: Parameters): ModuleValue[T] = {
+    val dftSink = dftNode.makeSink()
+    InModuleBody { dontTouch(dftSink.bundle) }
+  }
+}
+
+trait CanHaveDFT { 
```

Comment:
Is there a reason there are two traits for this? Can we combine them into one trait?

Reply:
trait HasDFT is use for IP block. ex) LPDDR4

trait CanHaveDFT is use for System Top. ex) SesameSystem

So, SesameSystem is just mixin CanHaveDFT trait.
Then automatically collect all DFT node and makeSink to System Top

Reply:
@jadekim727 , if `HasDFT` is a block-level trait and `CanHaveDFT` is a System-level trait, can we change `HasDFT` to `HasDeviceDFTPorts`? I think it needs to be more clear which trait is mixed in at the block-level vs which trait is mixed in at the System level